examples/manipulating_visualizing/plot_neurovault_meta_analysis.py

```diff
@@ -0,0 +1,99 @@
+"""
+NeuroVault meta-analysis of stop-go paradigm studies.
+=====================================================================
+
+This example shows how to download statistical maps from
+NeuroVault
+
+See :func:`nilearn.datasets.fetch_neurovault` documentation for more details.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+from nilearn import datasets
+
+### Define filters ############################################################
+cfilts = []  # [lambda col: col['DOI'] is not None]
+imfilts = [lambda im: "stop signal" in (im.get('cognitive_paradigm_cogatlas') or "").lower(),
```

Comment:
Using the ' or "" ', are you trying to use the pattern '.get(key, default)' ?

I would prefer to rely on that, because it is a standard Python pattern.